hummingbot/connector/exchange/bloxroute_openbook/bloxroute_openbook_order_data_manager.py

```diff
@@ -112,27 +44,24 @@ async def ready(self):
 
     @property
     def is_ready(self):
-        return self._is_ready
-
-    @property
-    def started(self):
-        return self._started
+        return self._ready.is_set()
 
     async def start(self):
-        if not self._started:
-            self._started = True
+        await self._start_lock.acquire()
+
+        if not self._started.is_set():
+            self._started.set()
+
             await self._initialize_order_books()
             self._orderbook_polling_task = asyncio.create_task(self._poll_order_book_updates())
 
             await self._initialize_order_status_streams()
 
-            await asyncio.sleep(self._start_timeout)
-
-            self._is_ready = True
             self._ready.set()
 
+        self._start_lock.release()
```

Comment:
probably should do this in a `finally` block in case of exceptions

Reply:
I just removed the lock, but I will keep this in mind